rhtap/summary.sh

```diff
@@ -34,6 +34,21 @@ function cosignTree() {
     cosign tree $URL
 }
 
+echo "SUMMARY_EYECATCHER_BEGIN"
+
+jq -n --arg base "$BASE_RESULTS" '
+  [ inputs ]' < <(
+    find "$BASE_RESULTS" -mindepth 2 -maxdepth 2 -name STATUS | while read -r status_file; do
+        name=$(basename "$(dirname "$status_file")")
+        result=$(tr -d '\n' < "$status_file" | xargs | tr '[:upper:]' '[:lower:]')
+
+        jq -n --arg name "$name" --arg result "$result" \
+            '{name: $name, result: $result}'
+    done
+) | jq '{results: .}'
```

Comment:
If the `jq ... '[inputs]'` call is there just to collect inputs into an array, I think we can drop it and instead add the `-s` flag here

```
--slurp / -s:

Instead of running the filter for each JSON object in the input, read the entire input stream into a large array and run the filter just once.
```